danger-kotlin-library/src/main/kotlin/systems/danger/kotlin/GitKtx.kt

```diff
@@ -0,0 +1,59 @@
+package systems.danger.kotlin
+
+// extensions over [Git] object
+
+/**
+ * Changed lines in this MR
+ */
+val Git.changedLines: PullRequestChangedLines
+    get() {
+        // TODO replace with more generic way to process commands
+        val utils = Utils()
+        val additionDeletionPairs = commits.mapNotNull { it.sha }
+            .map { sha ->
+                computeChangedLinesForCommit(utils, sha)
```

Comment:
Given the list is ordered, wouldn’t make sense to pass directly first and last commit and get the diff between them? That would make it a lot more efficient

Reply:
Actually you're right. We get only list of commits that are included in this PR but I can get first and last^1 and it works as well. Rewritten in c22ea384100a297780be7b0499ba6687a407e0b6